app/src/main/java/com/internship/move/utils/Utils.kt

```diff
@@ -87,6 +90,27 @@ fun Exception.toErrorResponse(errorResponseDtoJsonAdapter: JsonAdapter<ErrorResp
         ErrorResponse(message.toString())
     }
 
+ fun getScooterAddress(scooterCoords: CoordinatesDTO, context:Context): String {
```

Comment:
There's a more appropriate approach for this kind of method and its usage, from an architecture point of view.
Let's discuss when you have some time 😁